src/common/helpers/utils.ts

```diff
@@ -84,4 +84,21 @@ export function generatePhysicalName(
   }
 
   return prefix.toLowerCase() + (lower ? allParts.toLowerCase() : allParts) + '-' + uniqueStackIdPart;
+}
+
+export const maximumLambdaMemorySizeContextItem = 'maximumLambdaMemorySize';
+export const recommendedMaximumLambdaMemorySize = 7076;
+export function lambdaMemorySizeLimiter(construct: IConstruct, requestedMemorySizeInMegabytes: number) {
+  const maximumLambaMemorySize = construct.node.tryGetContext(maximumLambdaMemorySizeContextItem) === undefined ?
```

Comment:
Can this change impact sample repo? because we are not setting maximumLambdaMemorySizeContextItem in context there yet!

Reply:
Unless the context is set, "no". The `7076` is currently the maximum memory size any Lambda is set within the constructs library.